dayforce_client/client.py

```diff
@@ -99,16 +101,20 @@ class DayforceSFTP(object):
     username: str = attr.ib()
     password: str = attr.ib(repr=False)
     port: int = attr.ib(default=22)
-    known_hosts: str = attr.ib(default='~/.ssh/known_hosts')
+    # the host key can be retrieved with ssh-keyscan (pass just the base64 encoded part of the line beginning with ssh-rsa)
+    host_key: str = attr.ib(default=None)
     disable_host_key_checking: bool = attr.ib(default=False)
 
     def __attrs_post_init__(self):
+        self.cnopts = pysftp.CnOpts()
```

Comment:
@bobzoller we should probably explicitly make `cnopts` an instance attribute as well even though it doesnt get set unit post init.

```python
cnopts: pysftp.CnOpts  = attr.ib(init=False)
```